Source/WebCore/style/AnchorPositionEvaluator.cpp

```diff
@@ -925,6 +976,47 @@ static CSSPropertyID flipVertical(CSSPropertyID propertyID)
     }
 }
 
+static CSSPropertyID flipStart(CSSPropertyID propertyID, WritingMode writingMode)
+{
+    auto logicalProperty = CSSProperty::unresolvePhysicalProperty(propertyID, writingMode);
+
+    auto flippedLogical = [&] {
```

Comment:
Is this something that would be useful to generate? (I’ve been touching that code recently so happy to add it if you think it would be).